orkes-redis-queues/src/main/java/io/orkes/conductor/mq/redis/QueueMonitor.java

```diff
@@ -42,6 +42,14 @@ public abstract class QueueMonitor {
 
     private int maxPollCount = 100;
 
+    public void setMaxPollCount(int maxPollCount) {
+        this.maxPollCount = maxPollCount;
```

Comment:
The ArrayBlockingQueue is being initalized in the constructor with maxPollCount. If you wish to make use of this, please propagate it.

Reply:
Thanks for catching that. Done.